packages/python/plotly/plotly/tests/test_optional/test_kaleido/test_kaleido.py

```diff
@@ -2,6 +2,13 @@
 import plotly.io.kaleido
 import sys
 from contextlib import contextmanager
+from io import BytesIO
+
+if sys.version_info >= (3, 4):
+    from pathlib import Path
+else:
```

Comment:
Let's remove this, as we're dropping support for 2.7 and 3.5 in the next Plotly.py release.